compiler/version_check.go

```diff
@@ -19,6 +21,9 @@ const GoVersion = 16
 // at goroot, and reports an error if it's not compatible
 // with this version of the GopherJS compiler.
 func CheckGoVersion(goroot string) error {
+	if nvc, err := strconv.ParseBool(os.Getenv("GOPHERJS_NO_VERSION_CHECK")); err == nil && nvc {
```

Comment:
Maybe the name can be simplified to `GOPHERJS_VERSION_CHECK` to avoid the double negation? Since the error is being checked, it'll be fine when the env var is unset since `ParseBool` returns a non-nil error on the empty string. Another option is to use a non-negated verb like `GOPHERJS_SKIP_VERSION_CHECK`.

Reply:
Good point, I prefer `GOPHERJS_SKIP_VERSION_CHECK`, since it seems a bit more self-descriptive. I'm not much concerned about its length, since it should be only rarely needed by people working on GopherJS itself.